src/enums/domain.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+export enum Domain {
+  DEM = 'DEM',
+  RASTER = 'RASTER',
+  _3D = '3D',
+}
```

Comment:
there is a correlation with **record type**.
Should think where to store such types